inloop/accounts/views.py

```diff
@@ -1,38 +1,21 @@
 from django.contrib import messages
-from django.contrib.auth import update_session_auth_hash
-from django.contrib.auth.forms import PasswordChangeForm, SetPasswordForm
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import (PasswordChangeDoneView, PasswordChangeView,
+                                       PasswordResetCompleteView, PasswordResetConfirmView,
+                                       PasswordResetDoneView, PasswordResetView)
 from django.http.response import HttpResponseRedirect
 from django.template.response import TemplateResponse
 from django.urls import reverse, reverse_lazy
-from django.views.generic import FormView, TemplateView, View
+from django.views.generic import TemplateView, View
 
 from constance import config
-from password_reset.views import Recover, RecoverDone, Reset, ResetDone
 from registration.backends.hmac.views import (ActivationView as HmacActivationView,
                                               RegistrationView as HmacRegistrationView)
 
 from inloop.accounts.forms import SignupForm, StudentDetailsForm, UserChangeForm
 from inloop.accounts.models import StudentDetails
 
 
-class PasswordChangeView(LoginRequiredMixin, FormView):
```

Comment:
Why did you remove this class? Can you verify that Django's `PasswordChangeView` mixes in `LoginRequiredMixin`?

Reply:
https://github.com/django/django/blob/0b8abd7cdf1a5bae96dd0640af10ad504f104d06/django/contrib/auth/views.py#L336

Django's `PasswordChangeView` redirects users who are not logged in back to the login page.